### PR TITLE
Add Mint-specific instructions

### PR DESCRIPTION
**Pull request by ValikVityaz**
* [ ] Read PR [documentation](http://ostis-dev.github.io/sc-machine/dev/pr/)
* [ ] Update changelog
* [ ] Update documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,29 +9,24 @@
...
 ```
 
-__Ubuntu 18.04__ and __Ubuntu 20.04__
+__Linux Mint 19__
 
 ```sh
-sudo apt install librocksdb-dev libglib2.0-dev qtbase5-dev libboost-system-dev libboost-filesystem-dev libboost-program-options-dev make cmake antlr gcc g++ llvm-7 libcurl4-openssl-dev libclang-7-dev libboost-python-dev
-```
-
-Install Python development package
```
Comment (ValikVityaz) [inline README.md:23]: Not required to do manually as it is installed as a dependency of libboost-python-dev.

**Comment by deniskoronchik** [comment]
Do not try to reuse some line in this document, let's do different sections:

## Build on Ubuntu

## Build on Mint

... and this section should contain full Mint instruction even some of parts will duplicate Ubuntu one. It will be easier for users and support

**Review by deniskoronchik — CHANGES_REQUESTED** [review]
More clean structure

**Review by deniskoronchik — APPROVED** [review]

**Comment by deniskoronchik** [comment]
Fixup review fixes commits into one: "Review fixes"

**Review by deniskoronchik — CHANGES_REQUESTED** [review]
Fixup

**Review by KovalM — APPROVED** [review]

**Review by deniskoronchik — APPROVED** [review]

**Pull request merged**